test/runtests.jl

```diff
@@ -251,6 +251,14 @@ end
 @test sprint(JSON.print, [NaN]) == "[null]"
 @test sprint(JSON.print, [Inf]) == "[null]"
 
+# check Nullables are printed correctly
+@test sprint(JSON.print, [Nullable{Int64}()]) == "[null]"
+@test sprint(JSON.print, [Nullable{Int64}(Int64(1))]) == "[1]"
+
+# check Chars
+@test json('a') == "\"a\""
+@test json('\\') == "\"\\\\\""
```

Comment:
Can we test the following too?

- a non-ASCII character
- a control character (like `\t`)